spec/model/answer/money_spec.rb

```diff
@@ -49,30 +49,63 @@
         expect(instance.humanize).to eql("£1,523.42")
       end
     end
+
     context "no pence" do
       let(:money_input) { '1523.00' }
       it "rounds down amounts of money correctly" do
         expect(instance.humanize).to eql("£1,523")
       end
     end
+
+    context "for value with a trailing dot" do
+      let(:money_input) { '1523.' }
+      it "ignores the trailing dot" do
+        expect(instance.humanize).to eql("£1,523")
+      end
+    end
   end
 
-  describe "errors" do
-    context "invalid formatting" do
+  describe "parsing" do
+    context "no digits" do
       let(:money_input) {"Loads'a'money"}
 
-      it "Has errors" do
+      it "raises an error" do
         expect(instance.error).to eql("Invalid format")
       end
     end
 
-    context "no input" do
+    context "empty input" do
       let(:money_input) { nil }
 
-      it "Has errors" do
+      it "raises an error" do
         expect(instance.error).to eql("Please answer this question")
       end
     end
+
+    context "float with a trailing dot" do
+      let(:money_input) { "200." }
+
+      it "raises no errors" do
+        expect(instance.error).to eql(nil)
```

Comment:
I was thinking along the lines of these tests: https://github.com/alphagov/smartdown/blob/tt_forgiving_floats/spec/model/answer/money_spec.rb#L9-L21 That would give us coverage of how the values are parsed, not just when they are/aren't, as without being tied to `humanize` or similar (good point)